src/Web/Server/Controllers/TagController.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.AspNetCore.Mvc;
+
+using MRA.AssetsManagement.Application.Features.Tags.Commands;
+using MRA.AssetsManagement.Application.Features.Tags.Queries;
+using MRA.AssetsManagement.Domain.Entities;
+
+namespace MRA.AssetsManagement.Web.Server.Controllers;
+
+public class TagController : ApiControllerBase
```

Comment:
We had discussion about putting ProducesResponseType to the actions. 
What is the status of this?

<img width="597" alt="image" src="https://github.com/srp-internships/MRA.AssetsManagement/assets/48281953/4a340bf8-c7c6-4ba2-b50a-f20c40fa1eda">

Reply:
In progress. I'm learning it.